charts/mattermost-operator/templates/mysql-operator/statefulset.yaml

```diff
@@ -0,0 +1,99 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ .Values.mysqlOperator.appName }}-operator
+  namespace: {{ .Values.mysqlOperator.appName }}-operator
+  labels:
+    app: {{ .Values.mysqlOperator.appName }}-operator
+    release: {{ .Values.mysqlOperator.appName }}-operator
+spec:
+  replicas: {{ .Values.mysqlOperator.replicas }}
+  serviceName: {{ .Values.mysqlOperator.appName }}-operator-orc
+  podManagementPolicy: Parallel
+  selector:
+    matchLabels:
+      app: {{ .Values.mysqlOperator.appName }}-operator
+      release: {{ .Values.mysqlOperator.appName }}-operator
+  template:
+    metadata:
+      labels:
+        app: {{ .Values.mysqlOperator.appName }}-operator
+        release: {{ .Values.mysqlOperator.appName }}-operator
+{{- if .Values.mysqlOperator.statefulSet.annotations }}
+      annotations:
+        {{- toYaml .Values.mysqlOperator.statefulSet.annotations | nindent 8 }}
+{{- end }}
+    spec:
+      serviceAccountName: {{ .Values.mysqlOperator.appName }}-operator
+      containers:
+        - name: operator
+          image: "{{ .Values.mysqlOperator.image.repository }}:{{ .Values.mysqlOperator.image.tag }}"
+          imagePullPolicy: "{{ .Values.mysqlOperator.image.pullPolicy }}"
+          env:
+            - name: ORC_TOPOLOGY_USER
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Values.mysqlOperator.appName }}-operator-orc
+                  key: TOPOLOGY_USER
+            - name: ORC_TOPOLOGY_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Values.mysqlOperator.appName }}-operator-orc
+                  key: TOPOLOGY_PASSWORD
+          args:
+            - --leader-election-namespace={{ .Values.mysqlOperator.appName }}-operator
```

Comment:
if there are more args which we can pass it will be nice to be dynamic

Reply:
Added ability to add extraargs ;)